docs/modules/release-notes/pages/2023/v-3-26-legacy-jan-07-2023.adoc

```diff
@@ -0,0 +1,29 @@
+= v3.26| Legacy January 7, 2023
+:navtitle: v3.26 Legacy January 7, 2023
+
+These release notes cover Legacy Kobiton 3.26. As a reminder, in our November release, we started using 4.0+ as the numbering for releases of the new Kobiton, while Legacy Kobiton will continue with numbering starting with 3.25 and above. If you are using the new Kobiton, see the v4.1 release notes.
+
+If you have any questions or would like to update your Kobiton system, please send us an email at support@kobiton.com or contact your Account Executive.
+
+== New iOS and Android support
+
+Legacy Kobiton 3.26 supports the following:
+
+* iOS 15.7.1
+* iOS 16.1.1
+* iOS 16.1.2: supported except changing timezone
+* iOS 16.2: supported except changing timezone
+We also tested and support the following device model as stated:
+
+* Pixel 7: manual testing supported
+
+== Webhooks for device state
+
+Admins can now setup webhooks to receive automatic updates of device status changes for Private and Local devices, such as when a device goes offline or online.
+//  TO DO: Learn more here (Need to make the webhook document)
+
+_Note: The webhook settings can be seen by admins of any org, but this feature only supports Private and Local devices. It will not report on Public devices._
```

Comment:
Un-italicize the **Notes** section. Instead using the following syntax for notes:

```
[NOTE]
Your note here.
```

You can read more about how AsciiDoc handles notes here: https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/#admonition-syntax

(P.S. we use **Example 2** under the Admonition Syntax section)